### ISSUE TITLE
Console apps don't get forwarded all inputs

### ISSUE THREAD

**Issue by Fang-** [urbit/arvo#615]
ie when I hit `tab` in a CLI app, that app never gets an event. Same for things like `home` and `end`.

**Comment by ohAitch** [comment]
yeah, tab goes through to /+hood-drum as Ctrl-I just fine, but there's no
provision for forwarding ctrl keys in the /-sole protocol
